### ISSUE TITLE
Extra colours in XTherion

### ISSUE THREAD

**Issue by vldgeorgiev** [vldgeorgiev/therion#1]
Suggested by Bruce
Rock set xth(gui,me,rockcolor) peru 
Make point entities adopt the same colours as the line entities, e.g. 

And maybe a group for water, say…
set xth(gui,me,watercolor) blue
and for textual items such as label/remark/arrow/station-name/altitude etc…
set xth(gui,me,labelcolor) black

**Comment by CaverBruce** [comment]
maybe also a group for speleothems, and a group for man made features such as ropes, anchors etc

**Comment by CaverBruce** [comment]
Perhaps a colour for keywords not recognised, ie typing errors

**Comment by CaverBruce** [comment]
...I am regularly tracking down typing errors in th2 files...

**Comment by vldgeorgiev** [comment]
> Perhaps a colour for keywords not recognised, ie typing errors

Yes, that will be useful. I will check to what extent the current parsing allows it. Probably it will be possible for incorrect object types like "line wrongtype", not sure about wrong parameters like "line wall -wrongparam".
